### ISSUE TITLE
Remove bitcode for ios?

### ISSUE THREAD

**Issue by mzhu22** [cjsmith/react-native-connect-iq-mobile-sdk#3]
Hi, thanks for making this library!

I used the SDK in a React Native app, working with Expo and EAS. I can successfully build the app for iOS, and make an archive. However, I cannot upload it to TestFlight/App Store because it fails validation with:

>Invalid Executable. The executable 'MyApp.app/Frameworks/ConnectIQ.framework/ConnectIQ' contains bitcode.

I'm wondering if you've run into this before, and if you have any ideas for removing bitcode from the iOS code? Thanks!

**Comment by cjsmith** [comment]
Hmm.  I think the quick fix would be to either run `xcrun -sdk iphoneos bitcode_strip -r YOUR_EXECUTABLE` or put something like this in your ios/Podfile or somehow strip out the bitcode

```
post_install do |installer|
     
    installer.pods_project.targets.each do |target|
      if target.name == 'ConnectIQ'
        `xcrun -sdk iphoneos bitcode_strip -r ios/Pods/ConnectIQ/ConnectIQ.xcframework/ios-arm64/ConnectIQ.framework/ConnectIQ -o ios/Pods/ConnectIQ/ConnectIQ.xcframework/ios-arm64/ConnectIQ.framework/ConnectIQ`
        `xcrun -sdk iphoneos bitcode_strip -r ios/Pods/ConnectIQ/ConnectIQ.xcframework/ios-arm64_x86_64-simulator/ConnectIQ.framework/ConnectIQ -o ios/Pods/ConnectIQ/ConnectIQ.xcframework/ios-arm64_x86_64-simulator/ConnectIQ.framework/ConnectIQ`
     end
end
```

I think the proper thing to do would be for me to update https://github.com/cjsmith/connectiq-companion-app-sdk-ios/blob/main/ConnectIQ.podspec. to do this in the upstream dependency.  

The even more proper thing would be to get the cocoapods support merged into Garmin's official [connectiq-companion-app-sdk-ios](https://github.com/garmin/connectiq-companion-app-sdk-ios) and have them do it there, or change the build settings there to not build with bitcode.  

I will try do to the proper fix when I have time.

**Comment by mzhu22** [comment]
Cool, thanks for the help!

**Comment by zrachlin** [comment]
@mzhu22 was wondering if you were able to upload the app to Test Flight/App Store after removing the bitcode? Also - very curious how you got this library to work with expo - did you have to do anything special? I just assumed you'd have to use a vanilla RN project.

**Comment by mzhu22** [comment]
@zrachlin Yup I was able to upload to TestFlight after removing the bitcode! I haven't tried publishing to the App Store yet.

I'm new to Expo, so what I did was:
1. Prebuild my app for iOS
2. Open it in XCode
3. Manually do all the steps in https://developer.garmin.com/connect-iq/core-topics/mobile-sdk-for-ios/ (editing Info.plist, changing the AppDelegate code).

For now, I've been manually doing that every time I rebuild. But, from my understanding of Expo config plugins, I believe all of that can be automated... but I haven't given that a shot yet.

Hope that helps!
